### ISSUE TITLE
Issue with copying a project

### ISSUE THREAD

**Issue by vk011** [Microsoft/vscode-spring-boot-dashboard#65]
Hi,
I have a simple spring boot project running fine by starting it from the spring-boot dashboard.

However, I copied that project folder and opened it in VS Code, but I cannot run it, I get the `Could not find or load main class` error.

In Eclipse, if I copied a project, I would have to change the context root, how can I achieve something similar here please? I need to change the reference somewhere but I don't know where?

**Comment by Eskibear** [comment]
I guess if it’s because some generated files are out of sync. You may try executing Java:Update Project configuration (Shift+Alt+U) from the command palette to reimport the project.

**Comment by vk011** [comment]
I guess so too, but executing Java:Update Project  doesn't help.

**Comment by vk011** [comment]
The issue seems to be not that a folder is copied, but that new packages are created and classes moved about in those packages.

In the original project folder (which has been working for 2 days now), I just created a new package, moved a class there (home class, "/"), and it wouldn't work. I get the `cant find or load main class`.

Then I "put everything back", moved the class where it was originally and deleted the new package, however it still won't work. Is it possible that only I have this problem?

This is ridiculous and it cannot be used seriously. Unfortunately it's back to crappy Eclipse.

**Comment by vk011** [comment]
Found the culprit
!
The problem is the `lombok` dependency, removed it and now it all works again (both projects).

I never had issues with lombok in Eclipse, any idea why it is causing problems and who to turn to to fix it?

**Comment by Eskibear** [comment]
Do you have a Lombok extension installed in your vscode? I’ve heard about its conflict with the Java extension. 
Cc @yaohaizh , any idea?

**Comment by vk011** [comment]
Yes I do. Well, I did :) I believe I've tested it thoroughly now, bottom line is - spring boot won't run with lombok added as a dependency and the lombok extension installed. 

Maybe it would run if lombok support was added manually as I read somewhere on github, but I am not bothered right now to test that, I've spent the whole morning battling this crap. I might try it later on and if I do I will let you know.
